content/departments/product-engineering/process/planning.md

```diff
@@ -0,0 +1,93 @@
+
+1. A diagnosis. This is an explanation of the nature of the challenge. A good diagnosis simplifies the often overwhelming complexity of reality by identifying certain aspects of the situation as being the critical ones.
+2. A guiding policy. This is an overall approach chosen to cope with or overcome the obstacles identified in the diagnosis.
+3. Coherent actions. These are the steps that are coordinated with one another to support the accomplishment of the guiding policy.
+
+Recommended reading:
+
+- Short article: [The perils of bad strategy](https://www.mckinsey.com/business-functions/strategy-and-corporate-finance/our-insights/the-perils-of-bad-strategy)
+- Book: [Good Strategy/Bad Strategy: The difference and why it matters](https://www.amazon.com/Good-Strategy-Bad-difference-matters/dp/1781256179/ref=sr_1_1?crid=1H0B1F3WWUL2S&keywords=good+strategy+bad+strategy&qid=1639180521&sprefix=good+strategy%2Caps%2C205&sr=8-1) by Richard Rumelt
+
+### Good plans are clear and concise
+
+A good plan contains:
+
+- A definition of success that:
+  - Is clear and succinct (so a variety of stakeholders can quickly understand what is important)
+  - Is solution agnostic (so the team has ownership and space to find and pursue the best solution)
+  - Is framed as a customer and/or business problem statement (so we are always working on the right things for our business)
+  - Can be unambiguously measured and evaluated (so there is no disagreement about whether we have achieved success)
+- A concrete set of deliverables we believe will solve the problem (so we can detect/manage dependencies and do capacity planning)
+
+This definition applies recursively and fractically at every layer of a plan (for example: company plan, product/eng department plan, org plan, team plan) for a given unit of time (year, quarter, week).
+
+### Planning is iterative
+
+Arriving at a good plan is an iterative process that is neither exclusively top-down nor exclusively bottom-up because there is a cyclical information dependency. Company level strategy is informed by what is working and not working about our business (context our teams have), and team plans are informed by what our company level strategy is (a decision ultimately owned by the CEO). Fundmentally, everyone on our team has context and insights that are valuable inputs into the planning process.
+
+In practice, both the top-down motion and bottoms-up motion happen concurrently, but we need to bootstrap the process to iteratively work through the information dependency. It is helpful to think about planning as having a `W` shape:
+![Planning shape](https://assets.proof.pub/2056/firstround/O6rbG1rNQ2emg8EPj1Xa_TKImage%200%20Leadership%20Frames.png)
+
+Source: [The Secret to a Great Planning Process — Lessons from Airbnb and Eventbrite](https://review.firstround.com/the-secret-to-a-great-planning-process-lessons-from-airbnb-and-eventbrite)
+
+### Building the delivery plan is part of the work
```

Comment:
This section sort of repeats the "Good plans are clear and concise" section which describes what a good definition of success is and how to define concrete deliverables. Maybe they could be combined?

Reply:
Agree, thanks for this feedback! Moving up.